app/tests/system/nightwatch-config.js

```diff
@@ -21,35 +22,40 @@ module.exports = {
     'test_settings': {
         'default': {
             'globals' : {
-                'waitForConditionTimeout' : 60000
+                'waitForConditionTimeout' : 60000,
+                'waitForConditionPollInterval': 500
             },
+            'end_session_on_fail': false,
             'launch_url': 'http://localhost:4723/wd/hub',
             'selenium_host': 'localhost',
             'selenium_port': 4723,
             'silent': true,
             'output': true,
             'screenshots': {
-                'enabled': false,
-                'path': ''
+                'enabled': true,
+                'path': '${CIRCLE_ARTIFACTS}',
+                'on_failure': true
             },
             'desiredCapabilities': {
-                'fullReset': false,
                 'app': appPath,
                 'platformName': 'iOS',
-                'platformVersion': '8.4',
+                'platformVersion': '9.2',
```

Comment:
As we build out more parts of the Astro ecosystem, testing being one of them, it'd be nice if we could pull up configuration like iOS version, Xcode version, and test device name to a higher-level so that it's easier to config across a project.  Could we depend on environment variables here to specify these values and then in the build script set those up?